README.md

```diff
@@ -129,6 +129,7 @@ See a git log for the highlighted ref by pressing <kbd>SPACE</kbd>
 | <kbd>n</kbd>                                                     | Jump to next search result                          |
 | <kbd>N</kbd>                                                     | Jump to previous search result                      |
 | <kbd>q</kbd>, <kbd>Ctrl</kbd>+<kbd>c</kbd>, <kbd>esc</kbd>       | Quit                                                |
+| <kbd>y</kbd>      | Copy highlighted branch name                                                		|
```

Comment:
Can you format this to be like the other lines?

Reply:
Can we also move this up between the command `enter` and `space` and follow the verbiage of "Copy highlighted item"?

Reply:
done.